agent/engine/metadataservice/acquire_metadata.go

```diff
@@ -0,0 +1,47 @@
+package metadataservice
+
+import (
+	"github.com/aws/amazon-ecs-agent/agent/api"
+	"github.com/aws/amazon-ecs-agent/agent/config"
+	docker "github.com/fsouza/go-dockerclient"
+)
+
+//Main metadata gathering function.
```

Comment:
The comment should start with the name of the item, per document [here](https://golang.org/doc/effective_go.html#commentary)